static/js/util/profile_edit.js

```diff
@@ -1,11 +1,12 @@
 // @flow
 import React from 'react';
 import _ from 'lodash';
-import R from 'ramda';
 import TextField from 'material-ui/TextField';
 import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton';
 import Checkbox from 'material-ui/Checkbox';
 import Geosuggest from 'react-geosuggest';
+import R from 'ramda';
```

Comment:
Out of curiosity, why are you moving this input further down the list? Seems like Lodash and Ramda should be grouped together.